contracts/oracle/ChainlinkOracleWrapper.sol

```diff
@@ -0,0 +1,79 @@
+pragma solidity ^0.6.0;
+pragma experimental ABIEncoderV2;
+
+import "./IOracle.sol";
+import "../refs/CoreRef.sol";
+import "../external/SafeMathCopy.sol";
+import "@chainlink/contracts/src/v0.6/interfaces/AggregatorV3Interface.sol";
+
+/// @title Chainlink oracle wrapper
+/// @author eswak
+/// @notice Reads a Chainlink oracle value & wrap it under the standard Fei oracle interface
+contract ChainlinkOracleWrapper is IOracle, CoreRef {
+    using Decimal for Decimal.D256;
+    using SafeMathCopy for uint256;
+
+    /// @notice the referenced chainlink oracle
+    AggregatorV3Interface public chainlinkOracle;
+    uint256 public oracleDecimalsNormalizer;
+
+    /// @notice multiply the Chainlink oracle value by this constant, for instance for decimal corrections
+    uint256 public k;
+    uint256 public BASIS_POINTS_GRANULARITY = 10_000;
+
+    /// @notice should this oracle return 1 / oraclePrice instead of oraclePrice ?
+    bool public reverse;
```

Comment:
Lets remove this reverse logic to keep it simple and rely on the OracleRef to invert if they want to